public/main.js

```diff
@@ -0,0 +1,246 @@
+/** Необходимые константы */
+const apiKey = "d1000af359abdc5da2cf58cb576e8c75";
+const getUrl = `http://ws.audioscrobbler.com/2.0/?method=tag.gettopalbums&tag=edm&api_key=d1000af359abdc5da2cf58cb576e8c75&format=json`;
```

Comment:
Странное имя для переменной
+ Это явно константы, которые во всех языках программирования именуются в верхнем регистре
Тут должно быть что-то вроде:
```suggestion
const URL= `http://ws.audioscrobbler.com/2.0/?method=tag.gettopalbums&tag=edm&api_key=d1000af359abdc5da2cf58cb576e8c75&format=json`;
```